{"uid":"215168269288c6ae","name":"test_change_order_status[canceled-api-18]","fullName":"tests.endpoints.test_orders#test_change_order_status","historyId":"64a9041c793eaa2bb9f015ab6b096f49","time":{"start":1733072343015,"stop":1733072343963,"duration":948},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.ApiClient.ApiClient object at 0x7f700704d1c0>\nstatus = 'canceled', x_task_value = 'api-18'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Change order status\")\n    @pytest.mark.parametrize(\"status, x_task_value\", [(\"canceled\", \"api-18\")])\n    def test_change_order_status(api_client, status, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_orders.py:60: ValidationError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1733072330979,"stop":1733072330981,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"base_url","time":{"start":1733072330981,"stop":1733072330981,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"setup_test","time":{"start":1733072330982,"stop":1733072333947,"duration":2965},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"worker_id","time":{"start":1733072330982,"stop":1733072330982,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"api_client","time":{"start":1733072330982,"stop":1733072330982,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.ApiClient.ApiClient object at 0x7f700704d1c0>\nstatus = 'canceled', x_task_value = 'api-18'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Change order status\")\n    @pytest.mark.parametrize(\"status, x_task_value\", [(\"canceled\", \"api-18\")])\n    def test_change_order_status(api_client, status, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_orders.py:60: ValidationError","steps":[],"attachments":[{"uid":"76b6a1b8fa0bed67","name":"log","source":"76b6a1b8fa0bed67.txt","type":"text/plain","size":318},{"uid":"fa042d1f46a80083","name":"stderr","source":"fa042d1f46a80083.txt","type":"text/plain","size":314}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"setup_test::0","time":{"start":1733072349094,"stop":1733072349094,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"story","value":"Change order status"},{"name":"feature","value":"Orders API"},{"name":"parentSuite","value":"tests.endpoints"},{"name":"suite","value":"test_orders"},{"name":"host","value":"fv-az883-496"},{"name":"thread","value":"2337-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.endpoints.test_orders"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"status","value":"'canceled'"},{"name":"x_task_value","value":"'api-18'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"215168269288c6ae.json","parameterValues":["'canceled'","'api-18'"]}